types/highland/index.d.ts

```diff
@@ -888,8 +889,28 @@ declare namespace Highland {
 		 * @param {Function} f - the function to handle an error
 		 * @api public
 		 */
-		stopOnError(f: (err: Error) => void): Stream<R>;
-
```

Comment:
Could you undo this de-dent and have the new methods line up with existing indentation?

Reply:
fix, looks like the indent are using spaces and sometime tabs...

Reply:
Oh that’s not nice, well spotted, I’ll keep fixing that in mind for a future PR